radixdlt/src/main/java/org/radix/api/services/AtomsService.java

```diff
@@ -155,6 +121,42 @@ public AtomsService(LedgerEntryStore store, SubmissionControl submissionControl,
 		});
 	}
 
+	public void start() {
+		io.reactivex.rxjava3.disposables.Disposable lastStoredAtomDisposable = engineStore.lastStoredAtom()
+			.observeOn(Schedulers.io())
```

Comment:
What is this line for?

Reply:
Setting the thread on which observations will occur on